src/Core/Domain/Order/QueryResult/OrderInvoiceAddressForViewing.php

```diff
@@ -26,6 +26,17 @@
 
 namespace PrestaShop\PrestaShop\Core\Domain\Order\QueryResult;
 
+use PrestaShop\PrestaShop\Adapter\Address\AddressFormatter;
+
+@trigger_error(
+    sprintf(
+        '%s is deprecated since version 1.7.7.5 and will be removed in the next major version. Use %s instead.',
+        OrderInvoiceAddressForViewing::class,
+        AddressFormatter::class
+    ),
+    E_USER_DEPRECATED
+);
+
```

Comment:
You also have to add a phpdoc string 
https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations